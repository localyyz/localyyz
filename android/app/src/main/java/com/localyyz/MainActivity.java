package com.localyyz;

import android.content.Intent;
import android.os.Bundle;

import com.facebook.react.ReactActivity;
import com.facebook.react.ReactActivityDelegate;

import javax.annotation.Nullable;

import io.branch.rnbranch.RNBranchModule;

public class MainActivity extends ReactActivity {

    /**
     * Returns the name of the main component registered from JavaScript.
     * This is used to schedule rendering of the component.
     */
    @Override
    protected String getMainComponentName() {
        return "localyyz";
    }

    @Override
    protected void onStart() {
        super.onStart();
        RNBranchModule.initSession(getIntent().getData(), this);
    }

    @Override
    public void onNewIntent(Intent intent) {
        setIntent(intent);
    }

    @Override
    public void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
        MainApplication.getCallbackManager().onActivityResult(requestCode, resultCode, data);
    }

    @Override
    protected ReactActivityDelegate createReactActivityDelegate() {
        return new ReactActivityDelegate(this, getMainComponentName()) {
            @Nullable
            @Override
            protected Bundle getLaunchOptions() {
                Bundle initialProps = new Bundle();
                initialProps.putString("API_URL", BuildConfig.API_URL);
                return initialProps;
            }
        };
    }

}
