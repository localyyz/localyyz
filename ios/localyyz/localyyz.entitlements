<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>aps-environment</key>
	<string>development</string>
	<key>com.apple.developer.associated-domains</key>
	<array>
		<string>applinks:localyyz.app.link</string>
		<string>applinks:localyyz-alternate.app.link</string>
		<string>applinks:localyyz.test-app.link</string>
		<string>applinks:localyyz-alternate.test-app.link</string>
	</array>
	<key>com.apple.developer.in-app-payments</key>
	<array>
		<string>merchant.com.localyyz.toronto</string>
	</array>
</dict>
</plist>
