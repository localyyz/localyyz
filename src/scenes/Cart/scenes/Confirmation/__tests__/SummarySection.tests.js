import React from "react";

// custom
import { renderWithLayout, findById } from "localyyz/tests";

// local
import ComponentWrapper from "../SummarySection";
const Component = ComponentWrapper.wrappedComponent || ComponentWrapper;

function createProps(extra) {
  return {
    shippingAddress: {
      hasError: false
    },
    navigation: {
      getScreenProps: () => ({
        close: jest.fn()
      }),
      navigate: jest.fn()
    },
    ...extra
  };
}

describe(Component.name, () => {
  it("should render properly to snapshot", () => {
    let props = createProps();
    let c = renderWithLayout(<Component {...props} />);
    let [instance, layout] = [c.getInstance(), c.toJSON()];

    expect(instance).toBeDefined();
    expect(layout).toMatchSnapshot();
  });

  it("should navigate properly", () => {
    let props = createProps();
    let c = renderWithLayout(<Component {...props} />);
    let instance = c.getInstance();

    // call all navigate methods
    instance.updateEmail();
    instance.updateShipping();
    instance.updatePayment();

    // expects
    expect(props.navigation.navigate.mock.calls[0][0]).toBe("EmailScene");
    expect(props.navigation.navigate.mock.calls[1][0]).toBe("ShippingScene");
    expect(props.navigation.navigate.mock.calls[2][0]).toBe("PaymentScene");
  });

  it("should display email", () => {
    let props = createProps({
      email: "test@test.com"
    });
    let c = renderWithLayout(<Component {...props} />);
    let layout = c.toJSON();

    // expects
    expect(findById(layout, "emailValue").children[0]).toBe(props.email);
    expect(layout).toMatchSnapshot();
  });

  it("should display address", () => {
    let props = createProps({
      shippingAddress: {
        address: "test"
      }
    });
    let c = renderWithLayout(<Component {...props} />);
    let layout = c.toJSON();

    // expects
    expect(findById(layout, "shippingValue").children[0]).toBe(
      props.shippingAddress.address
    );
    expect(layout).toMatchSnapshot();
  });

  it("should display payment", () => {
    let props = createProps({
      payment: "0000"
    });
    let c = renderWithLayout(<Component {...props} />);
    let layout = c.toJSON();

    // expects
    expect(findById(layout, "paymentValue").children[0]).toBe(
      `Ending in ${props.payment}`
    );
    expect(layout).toMatchSnapshot();
  });
});
