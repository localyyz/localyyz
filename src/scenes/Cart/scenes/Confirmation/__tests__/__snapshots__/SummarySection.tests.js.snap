// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SummarySection should display address 1`] = `
<View
  style={
    Object {
      "marginHorizontal": 15,
      "marginVertical": 7.5,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Emailing receipt to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="emailValue"
        />
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Shipping to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="shippingValue"
        >
          test
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Payment via
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgb(30, 30, 30)",
                "fontSize": 16,
              },
              Object {
                "marginRight": 7.5,
              },
              Object {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="paymentValue"
        >
          Ending in undefined
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`SummarySection should display email 1`] = `
<View
  style={
    Object {
      "marginHorizontal": 15,
      "marginVertical": 7.5,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Emailing receipt to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="emailValue"
        >
          test@test.com
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Shipping to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="shippingValue"
        >
          
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Payment via
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgb(30, 30, 30)",
                "fontSize": 16,
              },
              Object {
                "marginRight": 7.5,
              },
              Object {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="paymentValue"
        >
          Ending in undefined
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`SummarySection should display payment 1`] = `
<View
  style={
    Object {
      "marginHorizontal": 15,
      "marginVertical": 7.5,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Emailing receipt to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="emailValue"
        />
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Shipping to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="shippingValue"
        >
          
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Payment via
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgb(30, 30, 30)",
                "fontSize": 16,
              },
              Object {
                "marginRight": 7.5,
              },
              Object {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="paymentValue"
        >
          Ending in 0000
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`SummarySection should render properly to snapshot 1`] = `
<View
  style={
    Object {
      "marginHorizontal": 15,
      "marginVertical": 7.5,
    }
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Emailing receipt to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="emailValue"
        />
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Shipping to
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="shippingValue"
        >
          
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexBasis": 150,
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "paddingHorizontal": 15,
            "paddingVertical": 10,
          },
          undefined,
          false,
        ]
      }
    >
      <View
        style={
          Object {
            "marginRight": 15,
          }
        }
      >
        <View
          style={Object {}}
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "backgroundColor": "rgba(255, 255, 255, 0)",
                  "color": "rgb(30, 30, 30)",
                  "fontSize": 14,
                  "fontWeight": "300",
                },
                undefined,
              ]
            }
          >
            Payment via
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgb(30, 30, 30)",
                "fontSize": 16,
              },
              Object {
                "marginRight": 7.5,
              },
              Object {
                "fontFamily": "FontAwesome",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          numberOfLines={1}
          style={
            Object {
              "backgroundColor": "rgba(255, 255, 255, 0)",
              "color": "#000000",
              "fontSize": 14,
              "fontWeight": "600",
            }
          }
          testID="paymentValue"
        >
          Ending in undefined
        </Text>
        <View
          style={
            Object {
              "alignItems": "flex-end",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 16,
                },
                undefined,
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
